data_validation/state_manager.py

```diff
@@ -154,7 +203,7 @@ def _read_gcs_file(self, file_path: str) -> str:
         gcs_file_path = self._get_gcs_file_path(file_path)
         blob = self.gcs_bucket.get_blob(gcs_file_path)
 
-        return blob.download_as_string()
+        return blob.download_as_bytes()
```

Comment:
The `_read_file()` method on [line 103](https://github.com/GoogleCloudPlatform/professional-services-data-validator/blob/6b2614dcfae3ca67d85221a9ceab3881c5c6b30d/data_validation/state_manager.py#L103) expects a string from this method

Reply:
Changed function from string to bytes because `download_as_string` already used to return bytes [despite its name](https://github.com/googleapis/google-cloud-python/issues/9290), and is [deprecated](https://github.com/googleapis/python-storage/issues/504).